### PR TITLE
better error and warning

### PR DESCRIPTION
**Pull request by pYr0x**
closes https://github.com/donejs/cli/pull/16

### CODE REVIEW

**Comment by matthewp** [comment]
Would you mind rebasing this so we can get it merged? thanks.

**Comment by pYr0x** [comment]
@matthewp i resolved the conflicts, should be now able to merge...

**Comment by matthewp** [comment]
If you rebase with master CI should pass.

**Comment by pYr0x** [comment]
@matthewp the ci failed because of an timeout

```
DoneJS CLI tests utils runScript and runCommand:
     Error: timeout of 120000ms exceeded. Ensure the done() callback is being called in this test.
      at null.<anonymous> (C:\projects\cli\node_modules\mocha\lib\runnable.js:215:19)
```

**Comment by pYr0x** [comment]
@matthewp tests pass, please merge.

File: bin/donejs [hunk 1]
```diff
@@ -10,7 +10,21 @@ var isWindows = require('os').platform() === 'win32';
...
 var log = utils.log;
 
-var mypkg = require(path.join(__dirname, '..', 'package.json'));
+var mypkg = {};
+try {
+  var pkgFile = path.join(__dirname, '..', 'package.json');
+  // try to access the package.json file
+  fs.accessSync(pkgFile, fs.R_OK);
+  try {
+    mypkg = JSON.parse(pkgFile, 'utf8');
```
Comment (matthewp) [inline bin/donejs]: This doesn't look right to me. pkgFile is going to be the path, like `path/to/package.json`, not the contents of the file.

**Comment by matthewp** [comment]
I'm a little uncomfortable that this is passing!  We might need a test for `bin/donejs`

**Comment by pYr0x** [comment]
so what is to be done?

File: bin/donejs (hunk 1 above)
Comment (matthewp) [inline bin/donejs]: This could be `mypkg = require(pkgFile)` instead.

**Comment by pYr0x** [comment]
@m-mujica can we close this? since you refactor the cli module?

**Comment by matthewp** [comment]
Yeah, good call.

**Pull request closed without merging**